api-docs/paths/scim/member_index.json

```diff
@@ -134,7 +134,7 @@
                                 "type": "string"
                             },
                             "groups": {
-                                "type": "array"
+                                "$ref": "../../components/schemas/scim/group_list.json#/GroupList"
```

Comment:
This was a wild guess, I haven't actually checked the API code -- corrections appreciated 🥳